src/components/apps/launch/AppLaunchForm.js

```diff
@@ -194,16 +194,27 @@ const initValues = ({
 
             let value = defaultValue || "";
 
+            if (
+                paramType === constants.PARAM_TYPE.FILE_FOLDER_INPUT ||
+                paramType === constants.PARAM_TYPE.FILE_INPUT ||
+                paramType === constants.PARAM_TYPE.FOLDER_INPUT
+            ) {
+                value = defaultValue?.path || "";
+            }
+
             if (paramType === constants.PARAM_TYPE.MULTIFILE_SELECTOR) {
-                value = defaultValue || [];
+                value = defaultValue?.path || [];
             }
+
             if (paramType === constants.PARAM_TYPE.FLAG) {
                 value = defaultValue && defaultValue !== "false";
             }
```

Comment:
Just double checking. Is `false` a string here ?

Reply:
Yes, this value is sent as a string by the service.